cluster/gce/manifests/network-proxy.manifest

```diff
@@ -0,0 +1,79 @@
+{
+"apiVersion": "v1",
+"kind": "Pod",
+"metadata": {
+  "name":"network-proxy",
+  "namespace": "kube-system",
+  "annotations": {
+    "scheduler.alpha.kubernetes.io/critical-pod": "",
+    "seccomp.security.alpha.kubernetes.io/pod": "docker/default"
+  }
+},
+"spec":{
+"hostNetwork": true,
+"containers": [
+    {
+    "name": "network-proxy-container",
+    "image": "gcr.io/google-containers/proxy-server:v0.0.1",
+    "resources": {
+      "requests": {
+        "cpu": "{{ cpulimit }}"
+      }
+    },
+    "command": [
+              "/proxy-server"
+    ],
+    "args": [
+      {{params}}
+    ],
+    {{container_env}}
+    "livenessProbe": {
+      "httpGet": {
+        "scheme": "HTTP",
+        "host": "127.0.0.1",
+        "port": {{admin_port}},
+        "path": "/healthz"
+      },
+      "initialDelaySeconds": {{ liveness_probe_initial_delay }},
+      "timeoutSeconds": 60
+    },
+    "ports": [
+      { "name": "serverport",
+        "containerPort": {{ server_port }},
+        "hostPort": {{ server_port }}
+      },
+      { "name": "agentport",
+        "containerPort": {{ agent_port }},
+        "hostPort": {{ agent_port }}
+      },
+      { "name": "adminport",
```

Comment:
What does the admin port do? What kind of interfaces does it expose? How is authn/z handled?

Reply:
The primary intent of the admin port is listen on localhost and provide /healthz and /metrics capabilities. 
We *might* add things like pprof if needed to that port. 
The idea was that you would only have access to that port if you are on that VM.
The current reference implementation (https://github.com/kubernetes-sigs/apiserver-network-proxy/blob/master/cmd/proxy/main.go#L321) does not setup authn/z for that port.
However that is certainly something we can fix before it goes alpha.